### PR TITLE
Added translations for home page

### PR DESCRIPTION
**Pull request by shivankacker**
## Proposed Changes

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Locale/en/Common.json [hunk 1]
```diff
@@ -192,5 +192,9 @@
   "live": "Live",
   "discharged": "Discharged",
   "archived": "Archived",
-  "no_changes_made": "No changes made"
-}
\ No newline at end of file
+  "no_changes_made": "No changes made",
+  "are_you_sure_want_to_delete": "Are you sure you want to delete {{name}}?",
+  "oxygen_information": "Oxygen Information",
+  "deleted_successfuly": "{{name}} Deleted Successfuly",
```
Comment (nihal467) [inline src/Locale/en/Common.json]:
```suggestion
  "deleted_successfully": "{{name}} Deleted Successfully",
```

File: src/Components/Facility/FacilityHome.tsx [hunk 2]
```diff
@@ -88,7 +88,7 @@ export const FacilityHome = ({ facilityId }: Props) => {
       onResponse: ({ res }) => {
         if (res?.ok) {
           Notification.Success({
-            msg: "Facility deleted successfully",
+            msg: t("deleted_successfuly", { name: facilityData?.name }),
```
Comment (nihal467) [inline src/Components/Facility/FacilityHome.tsx]:
```suggestion
            msg: t("deleted_successfully", { name: facilityData?.name }),
```

File: src/Locale/en/Common.json [hunk 3]
```diff
@@ -192,5 +192,9 @@
   "live": "Live",
   "discharged": "Discharged",
   "archived": "Archived",
-  "no_changes_made": "No changes made"
-}
\ No newline at end of file
+  "no_changes_made": "No changes made",
+  "are_you_sure_want_to_delete": "Are you sure you want to delete {{name}}?",
+  "oxygen_information": "Oxygen Information",
+  "deleted_successfully": "{{name}} Deleted Successfully",
```
Comment (rithviknishad) [inline src/Locale/en/Common.json]: This is more appropriate right?

```suggestion
  "deleted_successfully": "{{name}} deleted successfully",
```

**Comment by nihal467** [comment]
LGTM

**Review by rithviknishad — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**